### ISSUE TITLE
The required libraries are not installed

### ISSUE THREAD

**Issue by PF00517** [Shirakumo/radiance#57]
How do I get started? According to the documentation I did a step by step download of the files and the problem with (radiance:startup) started.

CL_USER> (radiance:startup)
To load "i-verbose":
  Load 1 ASDF system:
    i-verbose
; Loading "i-verbose"
[package piping]..................................
[package dissect].................................
[package verbose].........
2022-11-19 08:00:32 [INFO ] <RADIANCE>: Starting up.
2022-11-19 08:00:32 [INFO ] <RADIANCE>: Ensuring prerequisites.
To load "i-hunchentoot":
  Load 1 ASDF system:
    i-hunchentoot
; Loading "i-hunchentoot"
..................................................
[package impl-specific-gray]......................
[package trivial-gray-streams]....................
[package chunga]..................................
[package cl-base64]...............................
[package cl-fad]..................................
[package path]....................................
[package flexi-streams]...........................
..................................................
..................................................
..................................................
[package cl+ssl/config]...........................
[package trivial-garbage].........................
[package split-sequence]..........................
[package usocket].................................
[package cl+ssl]
debugger invoked on a CFFI:LOAD-FOREIGN-LIBRARY-ERROR in thread
#<THREAD "main thread" RUNNING {234F0049}>:
  Unable to load any of the alternatives:
   ("libcrypto-1_1.dll" "libeay32.dll")

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY                        ] Try loading the foreign library again.
  1: [USE-VALUE                    ] Use another library instead.
  2: [TRY-RECOMPILING              ] Recompile reload and try loading it again
  3: [RETRY                        ] Retry
                                     loading FASL for #<CL-SOURCE-FILE "cl+ssl" "src" "reload">.
  4: [ACCEPT                       ] Continue, treating
                                     loading FASL for #<CL-SOURCE-FILE "cl+ssl" "src" "reload">
                                     as having been successful.
  5:                                 Retry ASDF operation.
  6: [CLEAR-CONFIGURATION-AND-RETRY] Retry ASDF operation after resetting the
                                     configuration.
  7:                                 Retry ASDF operation.
  8:                                 Retry ASDF operation after resetting the
                                     configuration.
  9: [ABORT                        ] Give up on "i-hunchentoot"
 10: [REGISTER-LOCAL-PROJECTS      ] Register local projects and try again.
 11:                                 Exit debugger, returning to top level.

(CFFI::FL-ERROR "Unable to load any of the alternatives:~%   ~S" ("libcrypto-1_1.dll" "libeay32.dll"))
   source: (ERROR 'LOAD-FOREIGN-LIBRARY-ERROR :FORMAT-CONTROL CONTROL
                  :FORMAT-ARGUMENTS ARGUMENTS)
0]

I don't know what to do with these unloadable libraries and how to install them or how to solve this problem.

**Comment by Shinmera** [comment]
You'll have to install OpenSSL on your Windows machine.

**Comment by PF00517** [comment]
I have openssl installed and PATH paths added, but that did not solve the problem. As I understood CL+SSL needs these libraries, which are already loaded through CFFI. In the first solution to the problem, I added the library paths to *foreign-library-directories*, which didn't help. The second action I tried via CL+SSL/CONFIG by setting paths to exported macros, which didn't help either. The logs indicated one of two problems: either `"%1 is not a valid Win32 application"` (I tried x32 and x64 libraries) or that it couldn't find this library. 

I don't know who to ask for help with this problem, so let it stay here, maybe someone will be able to help someday.
